frontend/src/views/Demo.vue

```diff
@@ -50,6 +55,30 @@ import {
         "Talos",
       ],
     };
+  },
+
+  methods: {
+    async fetch() {
+      console.log(await ClusterResourceService.Get({
```

Comment:
Fast and dirty demo of how it's going to be used in the frontend.

Probably need export `pathPrefix` settings to some global variable.
As we mount grpc at `/api` path we have to use this prefix everywhere.

Additionally, I don't 100% like that in Watch I use GVR (GroupVersionResource) and here I only figured out how to get the resource using GVK (GroupVersionKind) and again I couldn't find a way to convert Resource to Kind yet.